### DISCUSSION TITLE
navbar justify-content-start

### DISCUSSION THREAD

**Discussion by balrogmedia** [bootscore/bootscore#315] [Q&A] [answered]
Hello,

I think the navbar is set by default with class justify-content-start. Or at least is set like that on my site. But I want the links in the navbar to look evenly, so I tried to swap the class with justify-content-evenly.
By changing class nothing happens, even just by removing the class completely, nothing changes.
<img width="662" alt="Screenshot 2022-11-21 at 19 13 15" src="https://user-images.githubusercontent.com/98542232/203129650-9f43eaf3-d20b-4d20-8c0b-f656703f9a43.png">

any idea on how to get around it?

thanks

**Comment by crftwrk** [comment, marked as answer]
Hello,

take a look to the nav-walker in https://github.com/bootscore/bootscore/blob/main/header.php line `66`.

```php
<!-- Bootstrap 5 Nav Walker Main Menu -->
<?php
wp_nav_menu(array(
  'theme_location' => 'main-menu',
  'container' => false,
  'menu_class' => '',
  'fallback_cb' => '__return_false',
  'items_wrap' => '<ul id="bootscore-navbar" class="navbar-nav ms-auto %2$s">%3$s</ul>',
  'depth' => 2,
  'walker' => new bootstrap_5_wp_nav_menu_walker()
));
?>

```

 Navbar has class `ms-auto` which pushes content to the end. Remove class `ms-auto` and navbar will have flex-start by default. In your case, replace `ms-auto` with `w-100 justify-content-evenly` classes.

Your nav-walker should look like this:

```php
<!-- Bootstrap 5 Nav Walker Main Menu -->
<?php
wp_nav_menu(array(
  'theme_location' => 'main-menu',
  'container' => false,
  'menu_class' => '',
  'fallback_cb' => '__return_false',
  'items_wrap' => '<ul id="bootscore-navbar" class="navbar-nav w-100 justify-content-evenly %2$s">%3$s</ul>',
  'depth' => 2,
  'walker' => new bootstrap_5_wp_nav_menu_walker()
));
?>
```

Solved?

**Reply by balrogmedia** [reply]
thank you very much!
